src/_includes/assets/js/banner.js

```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', () => {
+  const bannerText = document.getElementById('banner-text');
+  const banner = `{% t 'banner', {
+      '<0>': '<span>',
+      '</0>': '</span>',
+      interpolation: {
+          escapeValue: false
+      }
+    } %}`;
+  const banner_auth = `{% t 'banner-authenticated-donor', {
+      '<0>': '<span>',
+      '</0>': '</span>',
+      interpolation: {
+          escapeValue: false
+      }
+    } %}`;
```

Comment:
We'll need to grab the banner anchor node here, and get the links for each locale as well. Something like this could work:

```suggestion
  const bannerAnchor = document.getElementById('banner');
  const bannerTextNode = document.getElementById('banner-text');
  const bannerDefaultText = `{% t 'banner.default', {
      '<0>': '<span>',
      '</0>': '</span>',
      interpolation: {
          escapeValue: false
      }
    } %}`;
  const bannerDefaultLink = `{% t 'links:banner.default' %}`
  const bannerAuthText = `{% t 'banner.authenticated', {
      '<0>': '<span>',
      '</0>': '</span>',
      interpolation: {
          escapeValue: false
      }
    } %}`;
  const bannerAuthLink = `{% t 'links:banner.authenticated' %}`;
```

The suggestion above assumes that the previous suggestion about nesting the different banner text translations and links in an object is accepted. Also, feel free to change the variable names and the ordering.